client/src/components/ProductDetails/ProductGallery/index.jsx

```diff
@@ -12,8 +12,17 @@ const ProductGallery = ({ styleInfo }) => {
       style={{ position: `${expanded ? 'absolute' : 'relative'}` }}
       onClick={() => setExpanded(e => !e)}
     >
-      <Thumbnails photoIndex={photoIndex} photos={styleInfo?.photos} styleName={styleInfo?.name} />
-      <Carousel photoIndex={photoIndex} photos={styleInfo?.photos} setPhotoIndex={setPhotoIndex} styleName={styleInfo?.name} />
+      <Thumbnails
+        photoIndex={photoIndex}
+        setPhotoIndex={setPhotoIndex}
```

Comment:
If there needs to be additional control over the setState I would think think so, otherwise I would just be wrapping a setState with an empty function.